applications_tests/gls_nitsche_navier_stokes_33/sphere.geo

```diff
@@ -0,0 +1,4 @@
+SetFactory("OpenCASCADE");
+Sphere(1) = {0, 0, 0, 0.5, -Pi/2, Pi/2, 2*Pi};
+Physical Surface(0) = {1};
+Physical Volume(0) = {1};
```

Comment:
I question if it is a good idea to keep the geo file or not in the application test, but it's such a small file, why not...

Reply:
I don't have a strong opinion on this. I kept it because if we ever need to remake the mesh it could be useful, but at the same time it will be in the history.

Reply:
No worries, keep the geo file. Your rationale is good here.

Reply:
Maybe put is in Lethe-utils?